source/reference/limitations.txt

```diff
@@ -171,3 +171,8 @@ System Collections
 ------------------
 
 .. include:: /includes/collections/behavior-system-collections.rst
+
+Document Filtering 
```

Comment:
Should this be somewhere beta only?

Reply:
I'll remove this from here if we want to minimize visibility in the general docs.